impress-org/givewp
# Feature: Add additional gateway notice

Resolves [GIVE-1092]

## Description



This PR adds a notice about additional payment gateways to the payment gateway block.

## Affects



Adds a notice in the inspector controls.

Dev note: the external link element cannot be added to the block card description directly, which only accepts a string. The implementation here is a workaround, which is documented.

## Visuals



| Inspector Controls for Payment Gateways block |
|---|
| ![image](https://github.com/user-attachments/assets/0b03a172-b468-494e-828c-a2108ddb3b45)|
| ![Screenshot from 2024-08-05 13-27-22](https://github.com/user-attachments/assets/e6ed0a45-0ad2-4739-ba88-7781f8364fa4) |

## Testing Instructions



Create a v3 donation form.
Go to the Build tab/screen.
Click on the Payment Gateways block.
Dismiss the notice.
Reload the form builder.

## Pre-review Checklist






[GIVE-1092]: https://stellarwp.atlassian.net/browse/GIVE-1092?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
